addons/mrp/tests/test_replenish.py

```diff
@@ -40,6 +40,28 @@ def test_mrp_delay(self):
             wizard3 = self._create_wizard(product, wh)
             self.assertEqual(fields.Datetime.from_string('2023-01-06 00:00:00'), wizard3.date_planned)
 
+    def test_mrp_orderpoint_leadtime(self):
+        self.warehouse = self.env.ref('stock.warehouse0')
+        route_manufacture = self.warehouse.manufacture_pull_id.route_id
+        self.warehouse.resupply_route_ids = route_manufacture
+        product_1 = self.env['product.product'].create({
+            'name': 'Cake',
+            'type': 'product',
+            'route_ids': [(6, 0, [route_manufacture.id])]
+        })
+
+        # setup orderpoint (reordering rule)
+        rr = self.env['stock.warehouse.orderpoint'].create({
+            'name': 'Cake RR',
+            'location_id': self.warehouse.lot_stock_id.id,
+            'product_id': product_1.id,
+            'product_min_qty': 0,
+            'product_max_qty': 5,
+        })
+
+        info = self.env['stock.replenishment.info'].create({'orderpoint_id': rr.id})
+        self.assertEqual("0 days", info.wh_replenishment_option_ids.lead_time)
```

Comment:
please put a delay on the rule to be sure `_get_rule` return a rule and the delay is read correctly